spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature 'User Post Index Page' do
```

Comment:
- [ ] I think we still have two scenarios to be tested 
 1. **_Tests that asserts you can see a section for pagination if there are more posts than fit on the view._**
 2. **_Tests that redirects to the post's show page when you click on a post._**
  Please kindly consider testing them so that we can cover all functionalities of our app :+1: